app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,38 @@
+.Home-Menu {
```

Comment:
We are using `suitcss` naming style so this is usually `ComponentName-subComponent` so `Home-menu`